samples/Nancy/Nancy.Client/Nancy.Client.xproj

```diff
@@ -12,9 +11,14 @@
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
     <OutputPath Condition="'$(OutputPath)'=='' ">..\..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
   </PropertyGroup>
-
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
```

Comment:
That change should have come in from the DNX_Migration_Fixes branch to address Visual Studio not producing binaries on build. Was that branch not merged into the codebase? That's the only reference I can find to those lines in particular.